src/richie/apps/courses/models/category.py

```diff
@@ -45,6 +45,24 @@ def __str__(self):
             title=self.extended_object.get_title(),
         )
 
+    def get_meta_category(self):
+        """
+        Get the meta category the current category falls under. Meta-categories are a special kind
+        of categories that are not linked to courses but help us organize categories into
+        different kinds that are handled separately.
+        """
+        # Shorcut to the category's page node
+        node = self.extended_object.node
+
+        return Category.objects.filter(
+            extended_object__publisher_is_draft=False,
```

Comment:
Second thoughts:
1) it is akward to always return the public version: we should probably return the draft meta category for a draft category, and the published meta category for a published category...
2) we should probably not return a public version if it is not published in any language. This would allow us to not index a published category placed below a meta category that is not published? something like what is done here: https://github.com/openfun/richie/blob/master/src/richie/apps/courses/models/course.py#L141

Reply:
FTR, point 2 is actually a more general issue that must be addressed in another PR: https://github.com/openfun/richie/issues/704